danger-kotlin-library/src/main/kotlin/com/danger/dangerkotlin/DangerResults.kt

```diff
@@ -30,7 +31,7 @@ internal data class DangerResults(var fails: Array<Violation>,
     }
 }
 
-internal class Meta {
-    val runtimeName = "Danger Kotlin"
-    val runtimeHref = "https://danger.systems"
-}
\ No newline at end of file
+internal data class Meta(
+    val runtimeName: String = "Danger Kotlin",
+    val runtimeHref: String = "https://danger.systems"
```

Comment:
here can be also an URI rather than a String, but is not important right now

Reply:
This has to be sent back to Danger js, is a static string, this is why I think could make sense like that